includes/Modules/Ads.php

```diff
@@ -219,13 +235,26 @@ protected function setup_settings() {
 	 * A module being connected means that all steps required as part of its activation are completed.
 	 *
 	 * @since 1.122.0
+	 * @since n.e.x.t Add additional check to account for paxConversionID and extCustomerID as well when feature flag is enabled.
 	 *
 	 * @return bool True if module is connected, false otherwise.
 	 */
 	public function is_connected() {
 		$options = $this->get_settings()->get();
 
-		return parent::is_connected() && ( ! empty( $options['conversionID'] ) || ! empty( $options['paxConversionID'] ) || ! empty( $options['extCustomerID'] ) );
+		if ( Feature_Flags::enabled( 'adsPax' ) ) {
+			if ( empty( $options['conversionID'] ) && empty( $options['paxConversionID'] ) && empty( $options['extCustomerID'] ) ) {
+				return false;
+			}
+
+			return parent::is_connected() && ( ! empty( $options['conversionID'] ) || ! empty( $options['paxConversionID'] ) || ! empty( $options['extCustomerID'] ) );
+		}
+
+		if ( empty( $options['conversionID'] ) ) {
+			return false;
+		}
+
+		return parent::is_connected() && ! empty( $options['conversionID'] );
```

Comment:
This is redundant because the condition is already checked just above.
```suggestion
		return parent::is_connected();
```